### PR TITLE
Bugfix/windows 10 firewall service

### PR DESCRIPTION
**Pull request by jlariono**
- Fixed: Assert-FirewallConfigurable fails on Windows 10 due to firewall service name change.
- Fixed: ReparsePointTestFixture test was broken due to using TEMP environment variable which contains a shortened tilde path.

### CODE REVIEW

File: Carbon/Functions/Assert-FirewallConfigurable.ps1 [hunk 1]
```diff
@@ -35,7 +35,7 @@ function Assert-FirewallConfigurable
 
     Use-CallerPreference -Cmdlet $PSCmdlet -Session $ExecutionContext.SessionState
 
-    if( (Get-Service 'Windows Firewall').Status -ne 'Running' )
```
Comment (splatteredbits) [inline Carbon/Functions/Assert-FirewallConfigurable.ps1]: Keep this function backwards-compatible. Use both service names, please. Check for `Windows Firewall` first, then `MpsSvc`.

**Comment by jlariono** [comment]
@splatteredbits Requested changes made.

**Comment by jlariono** [comment]
Build failed due to exceeding maximum execution time (60min). The last successful build came in just under at 57min.

File: Carbon/Functions/Assert-FirewallConfigurable.ps1 [hunk 2]
```diff
@@ -35,11 +35,15 @@ function Assert-FirewallConfigurable
 
     Use-CallerPreference -Cmdlet $PSCmdlet -Session $ExecutionContext.SessionState
 
-    if( (Get-Service 'Windows Firewall').Status -ne 'Running' ) 
+    if( (Get-Service -DisplayName 'Windows Firewall' -ErrorAction Ignore).Status -eq 'Running' )
```
Comment (splatteredbits) [inline Carbon/Functions/Assert-FirewallConfigurable.ps1]: Take out the `-DisplayName` parameter so it is the same call as before. Carbon needs to  support all OSes that run PowerShell 4. This includes Windows 2008 R2. We don't have any 2008 R2 VMs to run our tests on, so this check has to stay *exactly* how it was, with the exception of the `-ErrorAction Ignore` parameter.

Comment (jlariono) [reply Carbon/Functions/Assert-FirewallConfigurable.ps1]: Oh gotcha, that makes much more sense now. I'll change it back.

**Comment by jlariono** [comment]
Build is now green, ready for merge.

**Pull request merged**